vividus-plugin-aws-s3/src/test/java/org/vividus/aws/s3/steps/S3BucketStepsTests.java

```diff
@@ -46,10 +48,12 @@
 
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({ S3BucketSteps.class, ObjectMetadata.class, IOUtils.class, ResourceUtils.class,
-        ByteArrayInputStream.class })
+        ByteArrayInputStream.class, FileUtils.class })
 @PowerMockIgnore("javax.management.*")
 public class S3BucketStepsTests
```

Comment:
why not to use Junit temp files extension and fully eliminate usage of power mock?

Reply:
Approach with power mock was previously approved